project-base/app/tests/FrontendApiBundle/Functional/Cart/ApplyPromoCodeToCartTest.php

```diff
@@ -78,7 +84,58 @@ public function testApplyPromoCode(): void
         $data = $this->getResponseDataForGraphQlType($response, 'ApplyPromoCodeToCart');
 
         self::assertEquals(CartDataFixture::CART_UUID, $data['uuid']);
-        self::assertEquals($promoCode->getCode(), $data['promoCodes'][0]['code']);
+        self::assertEquals(
+            [
+                'code' => $promoCode->getCode(),
+                'type' => 'percentage',
+                'discount' => [
+                    'priceWithVat' => '10.000000',
+                    'priceWithoutVat' => '10.000000',
+                    'vatAmount' => '0.000000',
```

Comment:
we should use `getFormattedMoneyAmountConvertedToDomainDefaultCurrency()` (or maybe better `getSerializedPriceConvertedToDomainDefaultCurrency()`) to assert the prices and keep the tests independent of domains setting changes on projects